### PR TITLE
Convert SchoolAdministratorDashboards.jsx to es6 stateless function

### PR DESCRIPTION
**Pull request by edavidsonsawyer**
# Who is this PR for?
Developers who want to contribute to the codebase by adding or improving client-side features.

# What problem does this PR fix?
Not all our JS files use ES6 modules, which can be confusing for new JS devs coming on to the project.

# What does this PR do?
Converts SchoolAdministratorDashboards.jsx to ES6 module

# Checklists

## Javascript QA

+ [x] Author checked latest in IE - School Administrator Dashboards Links
+ [x] Reviewer checked latest in IE - School Administrator Dashboards Links

### CODE REVIEW

**Comment by edavidsonsawyer** [comment]
@alexsoble here's the first of the es6 conversions. Since there are no handler methods or state here it's just the function, which seems to be the [typical standard](https://github.com/airbnb/javascript/tree/master/react#class-vs-reactcreateclass-vs-stateless). If we want to just make everything a class instead, let me know.

**Comment by alexsoble** [comment]
@edavidsonsawyer Woah, this is neat! I learned a new thing from you, thank you for sharing that Airbnb React style guide documentation! Let's change the file name too to match the CamelCasing standard we're moving towards.

File: app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx [hunk 1]
```diff
@@ -1,23 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (alexsoble) [inline app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx]: This looks unused.

Comment (edavidsonsawyer) [reply app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx]: It's used down in the propTypes variable to check the serializedData req. I could just use React.propTypes, but that's being depricated

**Comment by edavidsonsawyer** [comment]
@alexsoble sure thing, all set

**Comment by edavidsonsawyer** [comment]
@alexsoble well _now_ it's all set!

File: app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx (hunk 1 above)
Comment (alexsoble) [reply app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx]: Is `SchoolAdministratorDashboards` still a React component? It doesn't inherit from React.Component or React.createClass anymore, so it looks like just a plain function to me. That makes me wonder if `serializedData` is still props, or just a plain argument to a function now?

Comment (edavidsonsawyer) [reply app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx]: It is, actually! Anything that takes props as an argument and returns a React component is a valid component as far as React is concerned. https://reactjs.org/docs/components-and-props.html

Though you're right. It's also just a plain function with an argument. If we treat it like a react component we get type checking for the props and probably other stuff I'm just not thinking of atm.

Comment (alexsoble) [reply app/assets/javascripts/school_administrator_dashboard/dashboard_components/school_administrator_dashboards.jsx]: @edavidsonsawyer Ah I am learning so much here! This is cool!

**Pull request merged**